azure/group.sh

```diff
@@ -24,11 +24,11 @@ fn azure_group_get_names() {
 # azure_group_exists returns "0" if a resource group
 # already exists, "1" otherwise.
 fn azure_group_exists(name) {
-        groups <= azure_group_get_names()
-        for group in $groups {
-            if $group == $name {
-                return "0"
-            }
-        }
-        return "1"
+	group_name <= az group show --name $name | jq -r ".name"
```

Comment:
For this particular case `az group show` exists with `0` regardless if a group exists or not.

`nash` repl:
```console
λ> all <= azure_group_get_names()
λ> echo $all
xpto
λ> xpto_exists <= azure_group_exists("xpto")
λ> echo $xpto_exists
0
λ> ijk_exists <= azure_group_exists("ijk")
λ> echo $ijk_exists
1
```

`az` cli:
```console
$ az group show --name xpto
$ echo $?
// 0
$ az group show --name ijk
$ echo $?
// 0
```

Despite that I agree with you `klb` should be more reliable.